src/__snapshots__/index.test.tsx.snap

```diff
@@ -1,3 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`React Hook Form Input should render MUI Checkbox correctly 1`] = `[Function]`;
+
+exports[`React Hook Form Input should render MUI Radio correctly 1`] = `[Function]`;
+
+exports[`React Hook Form Input should render MUI Select correctly 1`] = `[Function]`;
+
+exports[`React Hook Form Input should render MUI Switch correctly 1`] = `[Function]`;
+
+exports[`React Hook Form Input should render MUI TextField correctly 1`] = `[Function]`;
```

Comment:
don't think those snapshot doing anything, they only output `Function`

Reply:
That was a terrible mistake I did there.

Reply:
it's my fault not yours lol